src/ServiceStack.OrmLite.SqlServer/SqlServer2012OrmLiteDialectProvider.cs

```diff
@@ -14,7 +14,7 @@ public class SqlServer2012OrmLiteDialectProvider : SqlServerOrmLiteDialectProvid
 
         public override bool DoesSequenceExist(IDbCommand dbCmd, string sequenceName)
         {
-            var sql = "SELECT EXISTS(SELECT 1 FROM SYS.SEQUENCES WHERE object_id=object_id({0}))"
+            var sql = "SELECT CASE WHEN EXISTS (SELECT 1 FROM SYS.SEQUENCES WHERE object_id=object_id({0})) THEN 1 ELSE 0 END"
```

Comment:
This will fail on a database w/ case-sensitive collation as the name of the system table is `sys.sequences` in lower case.

Reply:
thx for the pointer, updated in this commit https://github.com/ServiceStack/ServiceStack.OrmLite/commit/dadce014570bc02d943cb881e765f7e4fda6eaeb